app/models/govbox_set.rb

```diff
@@ -15,8 +15,8 @@ class GovboxSet
               :tweets,
               :video_news_items
 
-  def initialize(query, affiliate, geoip_info, options = {})
```

Comment:
`geoip_info` is not a location string. It's an instance of `GeoIP::City`, which is initialized in  the searches controller:
https://github.com/GSA/search-gov/blob/master/app/controllers/searches_controller.rb#L22

`@search = search_klass.new(@search_options.merge(geoip_info: GeoipLookup.lookup(request.remote_ip)))`

example:
```ruby
> g = GeoipLookup.lookup('67.170.115.191')
#<#<Class:0x00007fe27de17708>:GeoIP::City:0x7fe284879650
    area_code = 425,
    city_name = "Everett",
    continent_code = "NA",
    country_code2 = "US",
    country_code3 = "USA",
    country_name = "United States",
    dma_code = 819,
    ip = "67.170.115.191",
    latitude = 47.97900000000001,
    longitude = -122.2021,
    postal_code = "",
    real_region_name = "Washington",
    region_name = "WA",
    request = "67.170.115.191",
    timezone = "America/Los_Angeles"
>
```
So in order to pass a location string to USAJOBs, we'll need to extract it from that object. Something like:

```ruby
def location_name
  geoip_info = GeoipLookup.lookup(request.remote_ip))
  "#{geoip_info.city_name}, #{geoip_info.region_name}"
end
```
Or something like that. We should also verify the format that we need to pass to USAJOBs, such as for searchers in other countries, etc.